/**:
  ros__parameters:
    weight_param: 346
    height_param: 260
    focus: 6550.0
    pixel_size: 18.5
    events_topic: /event_camera/events
    imu_topic: /event_camera/imu
    count_image_topic: /count_image
    time_image_topic: /time_image
    time_image_vis_topic: /time_image_vis
    publish_time_image_vis: true
    use_event_order_time: false
    event_window_sec: 0.001
    log_time_stats: true
    enable_compensation: true
    publish_raw_count: true
    publish_comp_count: true
    raw_count_topic: /count_image_raw
    comp_count_topic: /count_image_comp
